bootstrap.ps1

```diff
@@ -0,0 +1,10 @@
+if ( !(Join-Path 'src' 'nimph.nim' | Test-Path) ) {
+  git clone git://github.com/disruptek/nimph.git
+  Set-Location nimph
+} 
+
+$env:NIMBLE_DIR = Join-Path $PWD 'deps'
```

Comment:
Again, ported from the original, but bugs me. Installing nimph to a local dir doesn't make much sense—you want the tool to be available everywhere after the installation.